README.md

```diff
@@ -63,7 +63,7 @@ For most transactions, `zwallet` uses the `0dns` to discover the network nodes,
 
 **Prerequisites**
 
-- go 1.13
+- Go: Installation instructions for Mac, LInux and Windows can be found here.
```

Comment:
`here` should be a link right?

Reply:
actually this point has to be removed the installation guides contains instructions for installing go.

now i have made a commit to update build instructions and remove this